web_server.js

```diff
@@ -52,6 +52,5 @@ io.on('connection', function (socket) {
 });
 
 server.listen(process.env.PORT || 80, () => {
-	console.log(`Server is listening on port: ${server.address().port}`);
-	log.info('simple-chat is starting...')
+	log.info(`Server is listening on port: ${server.address().port}`)
```

Comment:
Please add a semicolon at the end of the line. I know it's not necessary but I think it's a bad habit (since we're on serverside and minification doesn't really matter).

Reply:
Sure. and do you mind re-correct me if im wrong. Do you want this "log.info('simple-chat is starting...')" line to be on 8th line and "log.info(`Server is listening on port: ${server.address().port}`)" this to be on 55th yeah?

Reply:
That's correct. In line 55 the server is already running and waiting for requests, so the startup process is finished.

Reply:
Done.Please check the PR. :)